src/postamble.js

```diff
@@ -43,6 +43,9 @@ if (memoryInitializer) {
     memoryInitializer = Module['locateFile'](memoryInitializer);
   } else if (Module['memoryInitializerPrefixURL']) {
     memoryInitializer = Module['memoryInitializerPrefixURL'] + memoryInitializer;
+  } else if (ENVIRONMENT_IS_NODE) {
+    // Look for .mem file in the same folder as script by default
+    memoryInitializer = __dirname + '/' + memoryInitializer;
```

Comment:
I'm just saying that while overriding the current working directory might be a usable feature addition, that has nothing to do with this issue itself, as here the problem is exactly that Node.js (and only Node.js) API that is used - namely `fs.read*` - needs to be passed an absolute path to the `.mem` that is alongside the `.js` itself. APIs used for other runners already get this right, and the risk of breaking Node.js by fixing it is rather low, which is why it didn't seem very controversial to me.